xbmc/interfaces/builtins/PVRBuiltins.cpp

```diff
@@ -66,10 +67,12 @@ static int SeekPercentage(const std::vector<std::string>& params)
       CGUIInfoManager& infoMgr = CServiceBroker::GetGUI()->GetInfoManager();
 
       int iTimeshiftProgressDuration = 0;
-      infoMgr.GetInt(iTimeshiftProgressDuration, PVR_TIMESHIFT_PROGRESS_DURATION);
+      infoMgr.GetInt(iTimeshiftProgressDuration, PVR_TIMESHIFT_PROGRESS_DURATION,
```

Comment:
@ksooo @phunkyfish it'd be nice if calling GUI/InfoManager could be somehow avoided here (this info is probably available in the PVRManager itself. `EpgGridControl()` also seem to rely on GUI maybe the main logic can be also moved to `PVRGUIActions` without much effort?